MdeModulePkg/Include/Library/DeviceStateLib.h

```diff
@@ -0,0 +1,58 @@
+/** @file
+Functions used to support Getting and Setting Device States.
+
+Copyright (C) Microsoft Corporation. All rights reserved.
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+MU_CHANGE: new file
+**/
+
+#ifndef __DEVICE_STATE_LIB_H__

```

Comment:
```suggestion
#ifndef DEVICE_STATE_LIB_H_
```

This is been an ask on other PRs, to match style.